config/environments/development.rb

```diff
@@ -67,4 +67,16 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              'smtp.gmail.com',
+    port:                 587,
+    domain:               'gmail.com',
+    user_name:            'dhyeysapara5422',
+    password:             'ewrcgnjmexmpescr',
+    authentication:       'plain',
+    enable_starttls_auto: true,
+    open_timeout:         5,
+    read_timeout:         5 }
```

Comment:
We can't add credential into the github,
Please learn 'dot_env' gem and implement it into this PR.